spec/index.html

```diff
@@ -887,6 +889,23 @@ <h3>Reified Triples</h3>
       -->
     </pre>
 
+    <!-- sorry, I'm unsure how to refer to a specific example with a hyperlink -->
+    For instance, the syntactic sugar of the above example is equivalent to:
+    <pre id="ex-reified-triple-with-reifier"
```

Comment:
```suggestion
    <pre id="ex-reified-triple-with-reifier-expanded"
```

You can refer to an example using something like `<a href="#ex-reified-triple-with-reifier"></a>, which will fill in the text title for the example.